samples/excel/20-data-types/data-types-entity-icons.yaml

```
@@ -0,0 +1,118 @@
+order: 5
+id: excel-data-types-icons
+name: 'Data types: Create entity icons'
+description: Display all the icons available for entity data types.
+host: EXCEL
+api_set:
+    ExcelApi: '1.16'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#create-icons").click(() => tryCatch(createIcons));
+
+        // Retrieve the entity card icons enum.
+        const iconNames = Excel.EntityCompactLayoutIcons;
+        let icons;
+
+        function createEntitiesWithIcons(icons): Excel.EntityCellValue[][] {
+          // This method creates an entity data type for each
+          // icon in the `EntityCompactLayoutIcons` enum,
+          // and then displays the icon name with its icon.
```

Consider making this a multiline comment instead.
```suggestion
          /* This method creates an entity data type for each
           * icon in the `EntityCompactLayoutIcons` enum,
           * and then displays the icon name with its icon.
           */
```